packages/components/.storybook/webpack.config.js

```diff
@@ -5,38 +5,40 @@
 // IMPORTANT
 // When you add this file, we won't add the default configurations which is similar
 // to "React Create App". This only has babel loader to load JavaScript.
-const SASS_DATA = '@import "~bootstrap-talend-theme/src/theme/guidelines";';
+const SASS_DATA = "@import '~bootstrap-talend-theme/src/theme/guidelines';";
 const autoprefixer = require('autoprefixer');
 
-module.exports = {
-	module: {
-		loaders: [
-			{
-				test: /\.js?$/,
-				include: /node_modules\/(react-storybook-addon-props-combinations)/,
-				loader: 'babel',
-			},
-			{
-				test: /\.woff(2)?(\?[a-z0-9=&.]+)?$/,
-				loader: 'url-loader',
-				options: {
-					limit: 50000,
-					mimetype: 'application/font-woff',
-					name: './fonts/[name].[ext]',
-				},
-			},
-			{
-				test: /\.scss$/,
-				loaders: ['style', 'css?sourceMap&modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]', 'postcss', 'sass'],
+module.exports = (storybookBaseConfig) => {
+	storybookBaseConfig.plugins.pop();
```

Comment:
I suppose this is to remove Uglify, but this is not clear nor futureproof.
Can you replace it with a filter + affectation ?